tasks.py

```diff
@@ -298,3 +298,44 @@ def tests(context, name=NAME, image_ver=IMAGE_VER, local=INVOKE_LOCAL):
     pytest(context, name, image_ver, local)
 
     print("All tests have passed!")
+
+
+@task
+def html(context, sourcedir="docs/source", builddir="docs/build"):
+    """Creates html docs using sphinx-build command.
+
+    Args:
+        context (obj): Used to run specific commands
+        sourcedir (str, optional): Source directory for sphinx to use. Defaults to "source".
+        builddir (str, optional): Output directory for sphinx to use. Defaults to "build".
+    """
+    print("Building html documentation...")
+    clean_docs(context, builddir)
+    command = f"sphinx-build {sourcedir} {builddir}"
+    context.run(command)
+
+
+@task
+def api_doc(context, sourcedir="diffsync", output="docs/source/api"):
+    """Creates api docs using sphinx-apidoc command.
```

Comment:
Is this a task that needs to be run/rerun before running `html`? That's not immediately clear to me.

Reply:
This is an invoke tasks used prior to implementing sphinx-apidoc as a callback function to the sphinx build. I can remove this one.